### DISCUSSION TITLE
Need help installing TP-LINK T4U ver.3 driver

### DISCUSSION THREAD

**Discussion by justAguy321** [mocaccinoOS/mocaccino#108] [Q&A] [answered]
![Screenshot from 2022-10-12 21-43-22](https://user-images.githubusercontent.com/87280915/195487926-5210b451-c8bc-4b03-8097-a233e1fce3ee.png)
![Screenshot from 2022-10-12 21-44-18656](https://user-images.githubusercontent.com/87280915/195487928-8a9648e2-350a-470f-a5e2-d1aa4f0c2412.png)
![Screenshot from 2022-10-12 21-45-30](https://user-images.githubusercontent.com/87280915/195487929-5a776442-d3d0-4f6b-b803-0fdf87f383d2.png)
 the device shows up but making the file results in errors? any tips on getting this and other drivers working would be helpful.

**Comment by joostruis** [comment]
I think it could be as simple as this:

`luet install kernel-modules/rtl8812au-lts`

Then plugin the USB adapter again.
LTS here means that it is compiled for the default LTS kernel.

**Reply by joostruis** [reply]
Oops pardon me! This is still in staging repository! We will soon have this!
I will update this topic once it has landed in production repo. Expect it within 2 days.

**Reply by justAguy321** [reply]
Thanks, i appreciate your time and help 🙏

**Reply by joostruis** [reply]
Should be in the repository now.

**Comment by joostruis** [comment]
Can you let me know if the issue is resolved for you?

**Reply by justAguy321** [reply]
It installed but it still wont work. Can confirm the usb wifi card works on other devices. When i run lsusb it shows up but network wifi settings dont show it.

**Reply by msdobrescu** [reply]
When you plug it, what the logs say?

**Comment by joostruis** [comment]
This is my nano WiFi USB dongle that works with this driver:

`Bus 001 Device 006: ID 2357:011e TP-Link AC600 wireless Realtek RTL8811AU [Archer T2U Nano]`

**Comment by joostruis** [comment]
It seems you require a different package. rtl88x2bu 
I'll try to package that.

**Reply by justAguy321** [reply]
thanks, and sorry if i take a while to reply. i work on the road and sometimes im not home for a few days at a time.

**Comment by joostruis** [comment, marked as answer]
I just added kernel-modules/rtl88x2bu-lts can you try installing that?
